oak/client/policy_metadata.h

```diff
@@ -18,20 +18,27 @@
 #define OAK_CLIENT_POLICY_METADATA_H_
 
 #include "include/grpcpp/grpcpp.h"
+#include "oak/proto/policy.pb.h"
 
 namespace oak {
 
-// This class injects a pre-determined Oak Policy to each outgoing gRPC call.
+// This class injects the provided Oak Policy to each outgoing gRPC call, serialized as base64 and
```

Comment:
Is it still in base64?

Reply:
Well, it is behind the scenes, but that's now an implementation detail of gRPC so I removed it from here.